gordo_components/server/views/base.py

```diff
@@ -241,28 +311,6 @@ def _process_request(
                 f"Calculating model output took "
                 f"{get_model_output_time_s-process_request_start_time_s} s"
             )
-            transformed_model_input = model_io.get_transformed_input(
```

Comment:
Can probably remove these methods `model_io.get_transformed_input` & `get_inverse_transformed_input` 

Also should `get_transformed` be added later? Which just applies `transform` on the data? May be useful for folks to see that :man_shrugging: